kitsune/search/v2/documents.py

```diff
@@ -221,6 +221,20 @@ def get_field_value(self, field, instance, *args):
             instance = instance.question
         return super().get_field_value(field, instance, *args)
 
+    @classmethod
+    def prepare(cls, instance, **kwargs):
+        obj = super().prepare(instance, **kwargs)
+        # add a prefix to the id so we don't clash with QuestionDocuments
+        obj.meta.id = "a_{}".format(obj.meta.id)
+        return obj
+
+    @classmethod
+    def get(cls, id, **kwargs):
+        # if the id is un-prefixed, add it
+        if not str(id).startswith("a_"):
```

Comment:
It's a convenience, so we can continue to do something like:
```python
answer = Answer.objects.filter(foobar).first()
document = AnswerDocument.get(answer.id)
```
rather than
```python
document = AnswerDocument.get(f"a_{answer.id}")
```